Workiva/ixbrl-viewer
# Switch to write-excel-file for table export

#### Reason for change

See #416 - a large part of the viewer JS file was the exceljs library. 

#### Description of change

Replace the use of `exceljs` with the much smaller `write-excel-file`.  This change reduces the size of the production `ixbrlviewer.js` from 1.8MiB to 834KiB.

#### Steps to Test

Open a report with tables in it, and confirm that the "Export Table" output works as it did before. 

**review**:
@Workiva/xt
@paulwarren-wk
